source/assets/scripts/Router.js

```diff
@@ -35,14 +35,14 @@ export class Router{
         if(this[page] != undefined){
             let hash;
             if(page == 'home'){
-                hash = '';
+                hash = ' ';
```

Comment:
huh, interesting how this works over the other

Reply:
It seems like it wouldn't update the URL with an empty hash, but it will update it with space. Not sure why.